Mage/src/main/java/mage/abilities/dynamicvalue/DynamicValue.java

```diff
@@ -13,11 +14,75 @@
  */
 public interface DynamicValue extends Serializable, Copyable<DynamicValue> {
 
+    // Below are all the phrasings that different Effects employing the DynamicValue interface use in oracle text.
+    // The only impact these have on getMessage() is the plurality of the returned value.
+    // However, Effects will often need to cater to multiple phrasings to produce oracle text accurate to WOTC's whims.
+    enum EffectPhrasing {
```

Comment:
I would probably move this enum out to mage.constants.ValuePhrasing, unless you think it's cleaner to be nested